### PR TITLE
Add support for CAP_BPF and CAP_PERFMON privileges

### PR DESCRIPTION
**Pull request by felipecrs**
## Proposed change

Add support for setting the CAP_BPF and CAP_PERFMON capabilities.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/1759
- Link to cli pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [x] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

**Comment by felipecrs** [comment]
I'll craft the documentation PR as soon as I get a go ahead on this PR.

BTW this is a continuation of https://github.com/home-assistant/supervisor/pull/4130, which could not be done at that time because of the version of Docker.

**Review by agners — APPROVED** [review]

**Comment by felipecrs** [comment]
- Documentation PR: https://github.com/home-assistant/supervisor/pull/4259

File: supervisor/docker/const.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 class Capabilities(str, Enum):
     """Linux Capabilities."""
 
+    CAP_BPF = "CAP_BPF"
+    CAP_PERFMON = "CAP_PERFMON"
     DAC_READ_SEARCH = "DAC_READ_SEARCH"
     IPC_LOCK = "IPC_LOCK"
     NET_ADMIN = "NET_ADMIN"
```
Comment (agners) [inline supervisor/docker/const.py:12]: Also drop `CAP_` here, and order them by alphabet.

**Review by agners — CHANGES_REQUESTED** [review]

File: supervisor/addons/utils.py [hunk 2]
```diff
@@ -44,13 +44,15 @@ def rating_security(addon: AddonModel) -> int:
...
                 Capabilities.NET_RAW,
+                Capabilities.PERFMON,
                 Capabilities.SYS_ADMIN,
-                Capabilities.SYS_RAWIO,
-                Capabilities.SYS_PTRACE,
                 Capabilities.SYS_MODULE,
-                Capabilities.DAC_READ_SEARCH,
+                Capabilities.SYS_PTRACE,
+                Capabilities.SYS_RAWIO,
             )
```
Comment (felipecrs) [inline supervisor/addons/utils.py:56]: I just wonder why this list is missing SYS_TIME and SYS_RESOURCE, for example.

Comment (felipecrs) [reply supervisor/addons/utils.py:56]: Oh, now I understand.

Question: should we really decrease 1 point for BPF and PERFMON? Maybe not for PERFMON?

Comment (agners) [reply supervisor/addons/utils.py:56]: I'd say so yes. There is a reason for the `perf_event_paranoid` config, perf monitor has an attack surface... See e.g. https://patchwork.kernel.org/project/linux-hardening/patch/1469630746-32279-1-git-send-email-jeffv@google.com/.

**Review by agners — APPROVED** [review]

File: supervisor/addons/utils.py (hunk 2 above)
Comment (felipecrs) [reply supervisor/addons/utils.py:56]: Got it. Thanks.

**Review by frenck — APPROVED** [review]
Thanks, @felipecrs 👍 

../Frenck

**Pull request merged**